srv/params.go

```diff
@@ -132,7 +132,8 @@ func (p ParamsGetTransaction) IsValid() error {
 }
 
 type ParamsGetPegnetRates struct {
-	Height *uint32 `json:"height,omitempty"`
+	Height    *uint32 `json:"height,omitempty"`
+	Reference bool    `json:"reference,omitempty"`
```

Comment:
this doesn't seem to be used in https://github.com/pegnet/pegnetd/blob/2c8b741634c49355ed8bf4ed39f3cb5491249f0e/srv/methods.go#L409

Reply:
You are right. I meant to include that in this branch, but it's in the `larger_limit` branch which is rebased off this feature. That was my mistake. I can move it into this branch, but these branches will be shipped together. It would be easier to merge this as is, and get this param usage when `larger_limit` is also merged in.

https://github.com/pegnet/pegnetd/blob/larger_limit/srv/methods.go#L448

The Reference means to query for the pAsset rates vs the Asset rates. So currently only pUSD is returned as that is the only exchange rate recorded by the miners for pAssets.